### PR TITLE
Add a cron job to count content hits

### LINKED ISSUE #165: Create cronjob to run the hits counter on a weekly basis
## Summary

The hits counter script (`cnx-archive-hits_counter`) consumes the varnish log and counts the url hits for content. This information is then used to weight the search by popularity. The script needs to be run weekly to consume and update the data within the database.

## Environment

all

### PR DESCRIPTION
**Pull request by pumazi**
Closes #165

### CODE REVIEW

**Comment by pumazi** [comment]
`/etc/cron.daily/copy_varnish_logs` will be installed an run on servers that contain a varnish instance (e.g. staging{01,02,03}).

`/etc/cron.weekly/count_archive_hits` will be installed on the database server, which contains a copy of the archive codebase for triggers. This doesn't need to be run on the database server, but it seems to me that would the the easiest way to ensure there is only one instance of it out there, unless we can prevent a race condition from happening (maybe creating the timestamp file before processing the data and have a decent cleanup procedure).

**Comment by pumazi** [comment]
@DarkPrinceFrost The logic for command that gets call can be found at https://github.com/Connexions/cnx-archive/blob/master/cnxarchive/scripts/hits_counter.py Though it's probably not necessary to know the details, but just in case you wanted to know. 😃

**Comment by pumazi** [comment]
@dtwilliamson This is all yours to review and merge. 😄

File: roles/archive/templates/etc/cron.weekly/count_archive_hits [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#set -e
+
+log_root={{ varnish_logroot|default('/var/www/varnish_logs') }}
+log_user={{ varnish_user|default('varnishlog') }}
+log_group={{ varnish_group|default('varnish') }}
+combined_out="${log_root}/varnishncsa_combined.log"
+current_datestamp=$(date -d "$(printf '%(%m/%d/%Y)T 00:00:00')" +%s)
```
Comment (dtwilliamson) [inline roles/archive/templates/etc/cron.weekly/count_archive_hits]: date can do that by itself: `date -d '00:00:00' +%s` - you can add "today" if you want to make it more readable: `date -d 'today 00:00:00' +%s`.

File: roles/archive/templates/etc/cron.weekly/count_archive_hits [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#set -e
+
```
Comment (dtwilliamson) [inline roles/archive/templates/etc/cron.weekly/count_archive_hits]: I recommend always quoting variables by habit even if the values aren't expected to include whitespace.

**Pull request merged**